opennms-webapp-rest/src/test/java/org/opennms/web/rest/v2/MonitoringLocationRestServiceIT.java

```diff
@@ -72,7 +79,8 @@
 @JUnitTemporaryDatabase
 public class MonitoringLocationRestServiceIT extends AbstractSpringJerseyRestTestCase {
     private static final Logger LOG = LoggerFactory.getLogger(MonitoringLocationRestServiceIT.class);
-
+    private static final Integer DEFAULT_LIMIT = 10;
+    private static final Integer LOCATION_COUNT = 15;
```

Comment:
Just inline these. They are only relevant for one specific test.

Reply:
Done

Reply:
the lines are still here

Reply:
Removed them